src/Mercure/EventListener/AddLinkHeaderListener.php

```diff
@@ -43,26 +41,17 @@ public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFa
     public function onKernelResponse(ResponseEvent $event): void
     {
         $request = $event->getRequest();
-        // Prevent issues with NelmioCorsBundle
-        if ($this->isPreflightRequest($request)) {
```

Comment:
Why removing this?

Reply:
Discovery service already checks for this.